civihr_employee_portal/js/reports.js

```diff
@@ -792,11 +792,26 @@
           }
         });
       })
-      .controller('FiltersController', function () {
+      .controller('FiltersController', function (AbsencePeriod) {
         this.format = 'dd/MM/yyyy';
         this.placeholderFormat = 'dd/MM/yyyy';
         this.date = new Date();
         this.filtersCollapsed = true;
+      })
+      .service('AbsencePeriod', function () {
+        return {
+          getCurrent: function () {
+            var today = moment().format('YYYY-MM-DD');
```

Comment:
Should it be UTC?

Reply:
I based this implementation from the actual `AbsencePeriod` model and that one doesn't use UTC:

https://github.com/civicrm/civihr/blob/staging/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/absence-period.model.js#L42

I guess the current absence period is based on the local time.